src/merkle/store/mod.rs

```diff
@@ -87,6 +137,11 @@ impl MerkleStore {
     // PUBLIC ACCESSORS
     // --------------------------------------------------------------------------------------------
 
+    /// Return a count of the non-leaf nodes in the store.
+    pub fn qty_internal_nodes(&self) -> usize {
```

Comment:
nit: we usually prefix methods/variables which return number or count of something with `num`. So, I'd change this to `num_internal_nodes`.

Reply:
done :)